contracts/Randao.sol

```diff
@@ -34,17 +36,17 @@ contract Randao {
   address public founder;
 
   // Prevents methods from perfoming any value transfer
-  modifier noEther() { if (msg.value > 0) throw; _}
+  modifier noEther() { if (msg.value > 0) throw; _; }
 
```

Comment:
ah yes true, it's default now.
ok i'll break off 0.4 changes and send another pull request.
cheers.